Fix for cleaning old shopping carts when lifetime expired.

Bug confirmed present in version <=2.3.3, fixed in 2.3.4

Issue: https://github.com/magento/magento2/issues/24572
PR: https://github.com/magento/magento2/commit/2f04e666aceba9bcbd863b78613714922a1f0f35

--- a/Cron/CleanExpiredQuotes.php
+++ b/Cron/CleanExpiredQuotes.php
@@ -57,7 +57,6 @@ class CleanExpiredQuotes
 
             $quotes->addFieldToFilter('store_id', $storeId);
             $quotes->addFieldToFilter('updated_at', ['to' => date("Y-m-d", time() - $lifetime)]);
-            $quotes->addFieldToFilter('is_active', 0);
 
             foreach ($this->getExpireQuotesAdditionalFilterFields() as $field => $condition) {
                 $quotes->addFieldToFilter($field, $condition);
